DOCS OBSDOCS-922 Logging: Elastisearch removal notice (#73856)
openshift/openshift-docs: snippets/logging-elastic-dep-snip.adoc

[NOTE]
====

> The {logging-uc} 5.9 release does not contain an updated version of the {es-op}. If you currently use the {es-op} released with {logging-uc} 5.8, it will continue to work with {logging-uc} until the EOL of {logging-uc} 5.8. As an alternative to using the {es-op} to manage the default log storage, you can use the {loki-op}. For more information on the {logging-uc} lifecycle dates, read link:https://access.redhat.com/support/policy/updates/openshift_operators#platform-agnostic[Platform Agnostic Operators].

Reviewer: ```suggestion
The {logging-uc} 5.9 release does not contain an updated version of the {es-op}. If you currently use the {es-op} released with {logging-uc} 5.8, it will continue to work with {logging-uc} until the EOL of {logging-uc} 5.8. As an alternative to using the {es-op} to manage the default log storage, you can use the {loki-op}. For more information about the {logging-uc} lifecycle dates, read link:https://access.redhat.com/support/policy/updates/openshift_operators#platform-agnostic[Platform Agnostic Operators].
```

Reviewer: However:
```suggestion
The {logging-uc} 5.9 release does not contain an updated version of the {es-op}. If you currently use the {es-op} released with {logging-uc} 5.8, it will continue to work with {logging-uc} until the EOL of {logging-uc} 5.8. As an alternative to using the {es-op} to manage the default log storage, you can use the {loki-op}. For more information about the {logging-uc} lifecycle dates, see link:https://access.redhat.com/support/policy/updates/openshift_operators#platform-agnostic[Platform Agnostic Operators].
```

Author: Made this change. Thanks!
